src/react-components/profile-entry-panel.js

```diff
@@ -85,6 +93,69 @@ class ProfileEntryPanel extends Component {
   render() {
     const { formatMessage } = this.props.intl;
 
+    let panelBody;
+    switch (this.state.avatarType) {
+      case AVATAR_TYPES.LEGACY:
+        panelBody = (
+          <div className={styles.avatarSelectorContainer}>
+            <div className="loading-panel">
+              <div className="loader-wrap">
+                <div className="loader">
+                  <div className="loader-center" />
+                </div>
+              </div>
+            </div>
+            <iframe
+              className={styles.avatarSelector}
+              src={`/avatar-selector.html`}
+              ref={ifr => {
+                if (this.avatarSelector === ifr) return;
+                this.avatarSelector = ifr;
+                if (this.avatarSelector) {
+                  this.avatarSelector.onload = () => {
+                    this.sendAvatarStateToIframe();
+                  };
+                }
+              }}
+            />
+          </div>
+        );
+        break;
+      case AVATAR_TYPES.SKINNABLE:
+        panelBody = (
+          <AvatarEditor
+            signedIn={this.props.signedIn}
+            onSignIn={this.props.onSignIn}
+            onSignOut={this.props.onSignOut}
+            store={this.props.store}
+            onAvatarChanged={avatarId => this.setState({ avatarId })}
+            saveStateAndFinish={this.saveStateAndFinish}
+          />
+        );
+        break;
+      case AVATAR_TYPES.URL:
+        panelBody = (
+          <div className={styles.avatarSelectorContainer}>
+            <label htmlFor="#custom-avatar-url">Avatar GLTF/GLB </label>
+            <input
+              id="custom-avatar-url"
+              type="url"
+              required
+              className={styles.formFieldText}
+              value={this.state.avatarId}
+              onChange={e => this.setState({ avatarId: e.target.value })}
+            />
+            <div className={styles.info}>
+              Resources for creating custom avatar models can be found
+              <a target="_blank" rel="noopener noreferrer" href="https://github.com/j-conrad/hubs-avatar-pipelines">
```

Comment:
we should include a direct link to the blender file (with a blender logo probably) and eventually probably a link to a 1m tutorial